home/templates/profile.html

```diff
@@ -49,8 +64,15 @@ <h1 class="text-center my-4">Settings</h1>
 			$('.nav-tabs a[href="' + hash + '-noscroll"]').tab('show');
 		}
 
+		$("#professor-form-edit div.modal-lg").removeClass("modal-lg").addClass("modal-xl");
```

Comment:
rather than adjusting the modal size in js, the modal size should be customized in the form `EditReviewForm` class in python. Adjusting it here by depending on the ids and specific layout is fragile.

Reply:
The reason why I did it this way was because a crispy-forms modal doesn't allow customization of the modal size ([my fault](https://github.com/django-crispy-forms/django-crispy-forms/pull/1204) lol). It's set to a `modal-lg` and there's only 3 ways to change it. First way is the current approach. Second way is making a custom Modal template that is used for the `EditReviewForm` and passed as the template kwarg when creating the Modal in `professor_forms.py`. Third way is submitting a PR to change it on the crispy forms side, which is my preferred method. 

After looking into the state of crispy forms, it looks like they're now on V2 which contains breaking changes to V1 (our version). On top of that, V2 is having [several problems](https://github.com/django-crispy-forms/django-crispy-forms/issues) which I'm sure none of us want to deal with. So option 3 isn't really feasible in my opinion. 

Option 2 is doable and shouldn't be too bad because I recall how it works from when I made the bootstrap modal in the first place. Only issue with this option is that it will be another PR that will add another directory. It's gross but if we're ok with that then that's our best bet. 

If neither of those options work for us, then the current approach is literally the only option left.

Reply:
hmm, I can overlook this if the alternative is messing with crispy-forms internals.